### ISSUE TITLE
Error[BUG] 

### ISSUE THREAD

**Issue by zikehalil3535** [KaspianDev/AntiPopup#59]
### Describe the bug

`[17:39:14] [Server thread/ERROR]: Could not pass event PlayerJoinEvent to AntiPopup v6.1
java.util.NoSuchElementException: packet_handler
    at io.netty.channel.DefaultChannelPipeline.getContextOrDie(DefaultChannelPipeline.java:1073) ~[netty-transport-4.1.87.Final.jar:4.1.87.Final]
    at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:248) ~[netty-transport-4.1.87.Final.jar:4.1.87.Final]
    at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:237) ~[netty-transport-4.1.87.Final.jar:4.1.87.Final]
    at com.github.kaspiandev.antipopup.nms.v1_20.PlayerInjector_v1_20.inject(PlayerInjector_v1_20.java:63) ~[AntiPopup-6.1.jar:?]
    at com.github.kaspiandev.antipopup.nms.PlayerListener.onJoin(PlayerListener.java:18) ~[AntiPopup-6.1.jar:?]
    at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1319.execute(Unknown Source) ~[?:?]
    at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
    at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:git-Purpur-2017]
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
    at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:617) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]
    at net.minecraft.server.players.PlayerList.placeNewPlayer(PlayerList.java:339) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at net.minecraft.server.network.ServerLoginPacketListenerImpl.placeNewPlayer(ServerLoginPacketListenerImpl.java:202) ~[?:?]
    at net.minecraft.server.network.ServerLoginPacketListenerImpl.handleAcceptedLogin(ServerLoginPacketListenerImpl.java:183) ~[?:?]
    at net.minecraft.server.network.ServerLoginPacketListenerImpl.tick(ServerLoginPacketListenerImpl.java:85) ~[?:?]
    at net.minecraft.network.Connection.tick(Connection.java:611) ~[?:?]
    at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:234) ~[?:?]
    at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1616) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:487) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1421) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1192) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[purpur-1.20.1.jar:git-Purpur-2017]
    at java.lang.Thread.run(Thread.java:833) ~[?:?]`

### To Reproduce

If players join the server and then kicked getting error.

### Expected behavior

No Error

### Operating System

Windows 10

### Server Software

git-Purpur-2017 (MC: 1.20.1)

### Server Version

1.19.3

### Proxy Type

Velocity

### Plugin List

```bash
Plugins (89): AnarchyExploitFixes, AntiCheatAddition, AntiPopup, AntiVillagerLag, AuctionHouse, AureliumSkills, BankPlus, BattlePass, BestTools, BetterRTP, BloodInMinecraft, ChatControlRed, Citizens, ClansLite, ComplexTurrets, CraftSound, CrazyEnvoys, CustomNameplates, CustomPiglinsTrades, DeathMessages, DeluxeMenus, DiscordSRV, EZColors, Essentials, EssentialsChat, EternalCombat, ExcellentCrates, ExcellentEnchants, ExcellentShop, FastAsyncWorldEdit, GHomeGuiForEssentialsX, GSit, Hitmarkers, HotbarSounds, ImageFrame, InstantCommand, InteractionVisualizer, InvSeePlusPlus, ItemsAdder, KixsChatGames, LPX, LibsDisguises, LiteBans, LiteFish, LocketteX, LoneLibs, LuckPerms, LumberJack, MeteoritesPro, Multiverse-Core, MyCommand, MythicMobs, MythicTotem, NBTAPI, NexEngine, PinataParty, Pl-Hide-Pro, PlaceholderAPI, PlayerVaults, PocketHorses, ProtocolLib, RPGhuds, Rankup, RedstoneLimiter, RoseLoot, RoseStacker, SCore, ServerUtils, SetSpawn, SkinsRestorer, Skript, TAB, TempFly, TradeSystem, Vault, VelocityReport, VelocityVanish, ViaBackwards, ViaVersion, VillagerMarket, VillagerTradeLimiter, VirtualBooks, Votifier, Vulcan, WorldGuard, eGlow, sleep-most, spark and zVoteParty
```

### Server logs

.

**Comment by KaspianDev** [comment]
Please try with just AntiPopup installed and let me know if anything changed.

**Comment by zikehalil3535** [comment]
> Please try with just AntiPopup installed and let me know if anything changed.
> 
> Btw, logs say that 1.20 injector is used while you said that you are running 1.19.3, can you confirm the version with /version?

Yeah Im sorry, Im using 1.20 is there a problem?

**Comment by zikehalil3535** [comment]
> Please try with just AntiPopup installed and let me know if anything changed.
> 
> Btw, logs say that 1.20 injector is used while you said that you are running 1.19.3, can you confirm the version with /version?

For default server working fine.

**Comment by KaspianDev** [comment]
Add your plugins back one by one and see which one causes it.

**Comment by itsTyrion** [comment]
I've seen these issues occur with reloading. Did you reload the server or reload individual plugins using plugman or ServerUtils?

**Comment by KaspianDev** [comment]
> I've seen these issues occur out of the blue with reloading (not with this plugin). Did you reload the server or reload individual plugins using `/reload`, `Plugman`/`PlugmanX` or `ServerUtils`?

Antipopup is a packet based plugin, and packet plugins tend to break when reloaded so i wouldnt be suprised if that was the case.

**Comment by itsTyrion** [comment]
Yeah. I'm still wondering why this exact exception occurs. I faced it in a personal plugin. `NoSuchElementException: packet_handle` with a println showing packet_handle 1 line above so it *shouldn't* happen.

**Comment by NewPixelDev** [comment]
+1, this happened to me too

**Comment by KaspianDev** [comment]
I have no idea but if somebody has a clue please reopen the issue.
